bin/index.js

```diff
@@ -80,17 +89,26 @@ program
 
 function initCommand(options) {
   const selectedTemplate = options.template || "basic"; // Default to 'basic' if no template is specified
+  const packageName = options.name || "quick-start-express-server"; // Default to 'quick-start-express-server' if no name is specified
 
   if (!templates[selectedTemplate]) {
-    console.error(chalk.bgRed.white(`Template ${selectedTemplate} does not exist. To see available templates use "qse list".`));
+    console.error(
+      chalk.bgRed.white(
+        `Template ${selectedTemplate} does not exist. To see available templates use "qse list".`
+      )
```

Comment:
I see that you've worked on improving this log message. Instead of bg-red. Can we have text color to be red. Like highlight the template name that they provided and the error (red) and the `qse list`(yellow).

The bg red kinda is not legible.

![image](https://github.com/user-attachments/assets/1e63e8af-7c84-45bd-8ba5-cee5ea120790)

Reply:
Actually i never edited the log message. i had prettier code formatter locally set up. so it changed the single line code to multiple lines for better readability. that's why you can see theses lines as changed.

But i have made the changes you requested.
It was an easy thing to do.